aries-site/src/data/wcag/datatable.json

```diff
@@ -0,0 +1,518 @@
+[
+  {
+    "rule": "1.1.1",
+    "label": "Non-text content",
+    "status": "failed",
+    "notes": "Header sort buttons and search need better a11y text"
+  },
+  {
+    "rule": "1.2.1",
+    "label": "Audio-only and Video-only (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.2",
+    "label": "Captions (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.3",
+    "label": "Audio Description or Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.4",
+    "label": "Captions (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.5",
+    "label": "Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.6",
+    "label": "Sign Language (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.7",
+    "label": "Extended Audio Description (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.8",
+    "label": "Media Alternative (Prerecorded)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.2.9",
+    "label": "Audio-only (Live)",
+    "status": "not-applicable",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.1",
+    "label": "Info and Relationships",
+    "status": "failed",
+    "notes": "sortable and searchable headings need improvement"
+  },
+  {
+    "rule": "1.3.2",
+    "label": "Meaningful Sequence",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.3",
+    "label": "Sensory Characteristics",
+    "status": "failed",
+    "notes": "sortable and searchable headings need improvement"
+  },
+  {
+    "rule": "1.3.4",
+    "label": "Orientation",
+    "status": "passed",
+    "notes": ""
+  },
+  {
+    "rule": "1.3.5",
+    "label": "Identify Input Purpose",
+    "status": "passed",
```

Comment:
Not sure that this is applicable and instead identify purpose should be marked as applicable

Reply:
I changed I wonder if we got these mixed up

Reply:
DataTable itself isn't a form field but wouldn't the Search input you can get in a column header (when you click the search button) fall under this?

Reply:
Hmm good question, I feel like a search input doesn't fall into the category of input purposes outlined here: https://www.w3.org/TR/WCAG22/#input-purposes. 

The documentation also mentions: 

> The following input control purposes are intended to relate to the user of the content and pertain only to information related to that individual.

Which feels like it isn't applicable to a search component but rather a regular input like textinput.